addons/website/static/src/snippets/s_image_gallery/002.scss

```diff
@@ -132,6 +132,18 @@
         .carousel-indicators {
             align-items: center;
             min-width: 0;
+            overflow: auto;
```

Comment:
Behaviour: I think it would be nice if the numbers could scroll to, given that all the other indicator do :slightly_smiling_face: (But maybe we want to see all the numbers to see how many they are ? :thinking:) 

<img width="768" height="74" alt="image" src="https://github.com/user-attachments/assets/f6ae0946-a3f5-412e-b0c5-91a6a1716762" />